coordinators/build.gradle

```diff
@@ -17,6 +17,7 @@ android {
 
 dependencies {
   compile 'com.android.support:support-annotations:23.3.0'
+  compile 'com.android.support:support-v4:23.3.0'
```

Comment:
Added for the `isAttachedToWindow` call in `bind`. Could be convinced to inline that rather than take on the dep, no strong feelings either way.